apps/minifront/src/state/status.ts

```diff
@@ -10,9 +10,15 @@ export interface StatusSlice {
 
 export const createStatusSlice = (): SliceCreator<StatusSlice> => set => ({
   start: async () => {
-    const stream = viewClient.statusStream({});
-
     try {
+      //set the current status using unary request, since statusStream does not guarantee that we will get the status
+      const status = await viewClient.status({});
```

Comment:
Suggestion, but maybe this is an overoptimization, up to you:

```typescript

type Setter = (arg: (state: WritableDraft<AllSlices>) => void) => void;

const setStatusInState = (set: Setter, status: StatusResponse | StatusStreamResponse) => {
  set(state => {
    state.status.fullSyncHeight = status.fullSyncHeight;
    state.status.latestKnownBlockHeight = status.fullSyncHeight;
  });
};

export const createStatusSlice = (): SliceCreator<StatusSlice> => set => ({
  start: async () => {
    try {
      //set the current status using unary request, since statusStream does not guarantee that we will get the status
      const status = await viewClient.status({});
      setStatusInState(set, status);

      const stream = viewClient.statusStream({});
      for await (const status of stream) {
        setStatusInState(set, status);
      }
    } catch (error) {
      set(state => {
        state.status.error = error;
      });
    }
  },
});

```

Reply:
Unlike `StatusStreamResponse` there is no `latestKnownBlockHeight` field in `StatusResponse` (which is strange really) 
Instead of 
```
state.status.latestKnownBlockHeight = status.fullSyncHeight;
```
we need 
```
state.status.latestKnownBlockHeight = status.latestKnownBlockHeight;
```
And that requires checking that status `instanceof StatusStreamResponse`, so I'll just leave it as it is